### DISCUSSION TITLE
百度网盘账号无法在 docker-compose 中运行，errno: -9, refer to https://pan.baidu.com/union/doc/

### DISCUSSION THREAD

**Discussion by szerr** [AlistGo/alist#790] [Q&A]
使用 docker run 正常
docker run -d --restart=always -v /srv/volumes/alist:/opt/alist/data -p 127.0.0.1:5244:5244 --name="alist" xhofe/alist:latest

我的 docker-compose.yml：
version: '3'
services:
  cloudreve:
    image: xhofe/alist
    container_name: alist
    volumes:
      - /srv/volumes/alist:/opt/alist/data
    ports:
      - "127.0.0.1:5244:5244"
    restart: always

log：
alist        | [GIN] 2022/03/19 - 18:42:56 | 200 |      37.932µs |      172.26.0.1 | POST     "/api/admin/login"
alist        | [GIN] 2022/03/19 - 18:42:57 | 200 |      25.912µs |      172.26.0.1 | POST     "/api/admin/login"
alist        | [GIN] 2022/03/19 - 18:42:57 | 200 |      393.24µs |      172.26.0.1 | GET      "/api/admin/settings?group=0"
alist        | [GIN] 2022/03/19 - 18:42:59 | 200 |      29.887µs |      172.26.0.1 | POST     "/api/admin/login"
alist        | [GIN] 2022/03/19 - 18:42:59 | 200 |     490.595µs |      172.26.0.1 | GET      "/api/admin/accounts"
alist        | [GIN] 2022/03/19 - 18:42:59 | 200 |     538.897µs |      172.26.0.1 | GET      "/api/admin/drivers"
alist        | [GIN] 2022/03/19 - 18:43:26 | 200 |  204.725556ms |      172.26.0.1 | POST     "/api/admin/account/create"
alist        | [GIN] 2022/03/19 - 18:43:26 | 200 |     419.464µs |      172.26.0.1 | GET      "/api/admin/accounts"
alist        | [GIN] 2022/03/19 - 18:43:30 | 200 |      56.594µs |      172.26.0.1 | GET      "/"
alist        | [GIN] 2022/03/19 - 18:43:31 | 200 |      37.534µs |      172.26.0.1 | GET      "/api/admin/login"
alist        | [GIN] 2022/03/19 - 18:43:31 | 200 |     442.967µs |      172.26.0.1 | GET      "/api/public/settings"
alist        | ERRO[2022-03-19 18:43:32] errno: -9, refer to https://pan.baidu.com/union/doc/

**Comment by renjiu13** [comment]
我也遇到这个问题了

**Comment by HHHYEEE** [comment]
根目录要保持“/”，什么都不写
